schema/geometry_schema.json

```diff
@@ -444,6 +453,23 @@
 							"vertex_id"
 						]
 					}
+				},
+				"floor_to_ceiling_height_override": {
+					"description": "If provided, overrides the story floor_to_ceiling_height.",
+					"type": "number",
+					"minimum": 0,
+					"exclusiveMinimum": true
+				},
+				"floor_offset": {
```

Comment:
Using same naming logic, the floor offset could apply to the story (if you want floating multiplier stories), building (for the minimum z), and space (like it is here)